examples/oci-image-verification/go.mod

```diff
@@ -61,8 +63,8 @@ require (
 	github.com/sassoftware/relic v7.2.1+incompatible // indirect
 	github.com/secure-systems-lab/go-securesystemslib v0.9.0 // indirect
 	github.com/shibumi/go-pathspec v1.3.0 // indirect
-	github.com/sigstore/rekor v1.3.7 // indirect
-	github.com/sigstore/timestamp-authority v1.2.3 // indirect
+	github.com/sigstore/rekor v1.3.8 // indirect
+	github.com/sigstore/timestamp-authority v1.2.2 // indirect
```

Comment:
Was `timestamp-authority` downgraded by accident??

Reply:
No, it’s an indirect dep and has been downgraded elsewhere, presumably because the current version has a higher go directive. It’ll need similar changes to this repo to bring it in-line

Reply:
No, it’s an indirect dep and has been downgraded elsewhere, presumably because the current version has a higher go directive. It’ll need similar changes to this repo to bring it in-line

Reply:
Raised https://github.com/sigstore/timestamp-authority/pull/927 for that one

Reply:
Looking at the version tag comparison https://github.com/sigstore/timestamp-authority/compare/v1.2.2...v1.2.3 it seems to be 95% dependabot dependency bumps and only https://github.com/sigstore/timestamp-authority/pull/644 is a functional change, which is in CLI tooling that doesn't effect the import usage here, so I think it is fine for it to be downgraded in the meantime anyway

Reply:
We can cut a new release of timestamp-authority with the downgrade.

Reply:
Done - https://github.com/sigstore/timestamp-authority/releases/tag/v1.2.4

Reply:
Should this go mod file be updated as well?

Reply:
Yep fixed with new commit to do that and go mod tidy

Also bumped toolchain to latest go1.23.5 as that has been released in the interim too